powertools-sqs/src/main/java/software/amazon/lambda/powertools/sqs/SqsUtils.java

```diff
@@ -231,6 +231,10 @@ public static void overrideSqsClient(SqsClient client) {
                                              final SqsMessageHandler<R> handler) {
         final List<R> handlerReturn = new ArrayList<>();
 
+        if(client == null) {
```

Comment:
Lazy load SQS client, we dont want to initialise it beforehand since users might be overrriding it as well.

Reply:
We could improve the creation of the SqsClient?            

 .region(Region.of(System.getenv(SdkSystemSetting.AWS_REGION.environmentVariable())))

for example?

Reply:
Yeah its a possibilty, rite now I just converted to lazy load coz we have ability to specify overrriden client, want me to update ?

Reply:
I'm wondering if it might cause problems for local running.

Reply:
I believe it can, but the reason why I avoided optimising default client was that user always have an ability to override it to most optimized way but then again it might not be obvious for new users to do so

Reply:
yeah, ok.